src/mongoose_metrics.erl

```diff
@@ -162,6 +163,15 @@ get_dist_data_stats() ->
 get_up_time() ->
     {value, erlang:round(element(1, erlang:statistics(wall_clock))/1000)}.
 
+-spec tcp_ports() -> {value, integer()}.
+tcp_ports() ->
+    {value, length(port_list(name, "tcp_inet"))}.
+
+-spec port_list(Attr::atom(), term()) -> [port()].
+port_list(Attr, Val) ->
+    [Port || Port <- erlang:ports(),
+             {Attr, Val} =:= erlang:port_info(Port, Attr)].
```

Comment:
`port_info` crashes with a `badarg` if the port doesn't exist so there is a tiny window for a race condition here. It would be safer to add `catch` before it.